lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -598,7 +598,7 @@ def test_build_student_data_for_block_with_real_generate_report_data(self):
         student_data, _ = ProblemResponses._build_student_data(
             user_id=self.instructor.id,
             course_key=self.course.id,
-            usage_key_str=str(self.course.location),
+            usage_key_str_list=[str(self.course.location)],
```

Comment:
`usage_key_str_list` is not used in ironwood release (ref: https://github.com/edx/edx-platform/blob/441d6384eef5ed0b62e936a7d85958d10e43d92e/lms/djangoapps/instructor_task/tests/test_tasks_helper.py#L541)